### PR TITLE
16 - Change - stack / dash - stack to Palamar

### PR DESCRIPTION
**Pull request by sametbalta**

### CODE REVIEW

File: templates/index.html [hunk 1]
```diff
@@ -3,7 +3,7 @@
 <head>
   <meta charset="utf-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
-  <title>{% block title %}dashStack Dashboard{% endblock %}</title>
+  <title>{% block title %}palamar Dashboard{% endblock %}</title>
```
Comment (sametbalta) [inline templates/index.html]: I'm not sure about this title. It should start lower case, should not?

Comment (kelepirci) [reply templates/index.html]: It should start with capital P. Like Palamar.

Also we are using oslo_i18n. You should change the template accordingly.

**Review by sametbalta — COMMENTED** [review]
Title is updated

**Review by kelepirci — APPROVED** [review]
Good work Samet and congratz on your first commit.

**Pull request merged**